crates/edr_eth/src/remote/client.rs

```diff
@@ -1753,13 +1745,13 @@ mod tests {
                 .expect("failed to parse address");
 
             let error = TestRpcClient::new(&alchemy_url)
```

Comment:
I feel like we could just delete this test. As far as I understand, it's testing for a specific error response from Alchemy, which is liable to change.

Reply:
Do you mean to remove the whole test or the specific error?

I feel that the tests did provide value in this case, as some APIs changed from returning an error to returning an option. Having the tests, helped us catch that instead.

Reply:
Ah ok I didn't catch that 👍 I'd probably remove the check for a specific error code or message, but I'll leave it up to you.